src/chai-form-configurator.ts

```diff
@@ -0,0 +1,414 @@
+import { LitElement, html, css } from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+import { ChaiForm } from './chai-form';
+
+@customElement('chai-form-configurator')
+export class ChaiFormConfigurator extends LitElement {
+  @property({ type: String }) targetForm: string = '';
+
+  static override styles = css`
+   
+    .form-configurator-container
+    {
+      color: black;
+    padding: 20px;
+    background-color: #fbf7f1;
+    border-radius: 20px;
+    text-shadow: none;
+    font-weight: bolder;
+    font-size: 14px;
+    }
+
+    .row
+    {
+      margin-bottom:15px;
+      font-family:sans-serif;
+    }
+    .form-configurator-header-message
+    {
+      margin-bottom: 34px;
+    font-weight: bold;
+    }
+  `;
+
+  constructor() {
+    super();
+    this.targetForm = ''; 
+  }
+
+  @property({ type: String }) selectedVisibilityValue: string = '';
+
+
+  toggleChaiFormSlots(event : Event ,  componentName : string)
```

Comment:
Explain what this is doing with a comment.

Reply:
Another way of saying that is that the code doesn't make it clear what you're doing. What is an `isCreateName` attribute on the `chai-form` supposed to mean?